Addenda

curiousrohan1 opened the pull request:
addenda

curiousrohan1 committed:
Made some addenda, will continue tomorrow. Signing out for 6/22

curiousrohan1 committed:
Made some addenda, will continue tomorrow. Signing out for 6/22

curiousrohan1 committed:
a

stamhankar999 commented on src/main/resources/public/js/components/leftPane.js:
```diff
@@ -1,5 +1,5 @@
 let leftPane = app.component('Leftpane', {
-  emits: ['update-err'],
+  emits: ['update-err', 'focus','switch'],
```
Add a space between 'focus' and 'switch'.

stamhankar999 commented on src/main/resources/public/js/index.js:
```diff
@@ -155,6 +178,29 @@ const store = Vuex.createStore({
         }
       }
+      state.consolidatedIngredients.sort(
+        (a, b) => {
```
This comparator is exactly the same as the other one. This should be a named function (say `ingComparator`) and then you call sort like this: `state.consolidatedIngredients.sort(ingComparator);`

stamhankar999 commented on src/main/resources/public/style.css:
```diff
@@ -159,3 +159,9 @@ img {
+}
+table{
+    width:100%;
+}
```
Add newline at end of file; configure your IDE to do that for you.

curiousrohan1 committed:
*Added a new line at the end of style.css
*Added a space between the emits in leftPane.js
*Added ingComparator method to lessen redundancies regarding the sorting of ingredients in makeRec.js and recipes in makeGroc.js in index.js

curiousrohan1 committed:
*Removed switch in leftPane.js

curiousrohan1 committed:
* Increased the priority of the unit clause of the ingComparator above quantity
* Made the ingredient render in makeGrocery.js better, as per sandman's suggestions

curiousrohan1 committed:
* Change message in deletion of recipe modal in makeGrocery.js
